From 92a37bcb00b036a958af2cc955437b53cbdc37dc Mon Sep 17 00:00:00 2001
From: Nikolay Aleksandrov <nikolay@redhat.com>
Date: Mon, 1 Sep 2014 13:33:08 -0400
Subject: [redhat] Kconfig: Add qat on x86_64 kernel config option

Message-id: <1409578401-26943-17-git-send-email-nikolay@redhat.com>
Patchwork-id: 90094
O-Subject: [RHEL7.1 PATCH BZ 1091130 16/29] qat: Add x86_64 kernel config option
Bugzilla: 1091130
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Herbert Xu <herbert.xu@redhat.com>

RHEL-specific to add QAT's kernel config option for
CRYPTO_DEV_QAT_DH895xCC (x86_64).
When CRYPTO_DEV_QAT_DH895xCC is built, it automatically selects CRYPTO_DEV_QAT.

Signed-off-by: Nikolay Aleksandrov <nikolay@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/x86_64/CONFIG_CRYPTO_DEV_QAT_DH895xCC b/redhat/configs/generic/x86_64/CONFIG_CRYPTO_DEV_QAT_DH895xCC
new file mode 100644
index 0000000..3d37c7a
--- /dev/null
+++ b/redhat/configs/generic/x86_64/CONFIG_CRYPTO_DEV_QAT_DH895xCC
@@ -0,0 +1 @@
+CONFIG_CRYPTO_DEV_QAT_DH895xCC=m
-- 
1.7.1